src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -0,0 +1,715 @@
+//==============================================================================
+// Parameters
+//==============================================================================
+
+@description('Optional. Name of the hub. Used to ensure unique resource names. Default: "finops-hub".')
+param dataFactoryName string
+
+@description('Required. The name of the Azure Key Vault instance.')
+param keyVaultName string
+
+@description('Required. The name of the Azure storage account instance.')
+param storageAccountName string
+
+@description('Required. The name of the container where Cost Management data is exported.')
+param exportContainerName string
+
+@description('Required. The name of the container where normalized data is ingested.')
+param ingestionContainerName string
+
+@description('Optional. Indicates whether ingested data should be converted to Parquet. Default: true.')
+param convertToParquet bool = true
```

Comment:
I simplified the `datasetType` logic to just support opting in to converting to parquet. If we want to support more formats, we can always add them later, but the long-term goal is to get into a database, so I'm not too concerned.